x11-terms/terminator/terminator-1.92.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_6 python3_7 )
```

Comment:
please keep an empty line between EAPI and everything else. while we're at it, could you also check with py38 already? then you can shorten it with python3_{6,7,8}

Reply:
Sure. And because my python3.8 installation is so empty now, it revealed another issue here - missing `[${PYTHON_USEDEP}]`